Specification/BrAPI-Schema/BrAPI-Germplasm/PedigreeNode.json

```diff
@@ -124,9 +124,9 @@
                 "primaryModel": true
             }
         },
-        "Parents": {
+        "GermplasmParent": {
             "properties": {
-                "parentDbId": {
```

Comment:
This is interesting issue. @BrapiCoordinatorSelby parentDbId was added to the JSON schema but is not part of v2.1. It was added to the JSON schema as 'Parents', with the additional properties parentDbId (replacing germplasmId) and pedigreeNode. To be totally consistent with other links there should a germplasm property with a link to a germplasm. Perhaps the pedigreeNode can be added in v2.2? Also renaming 'Parents' to 'GermplasmParent' is good idea and does not affect the model, but it is clearly because the object is a single parent.

Reply:
I have removed the DbId again.
If I understand your comment correctly, it would otherwise be a model change again at the moment, wouldn't it?